src/urllib3/connection.py

```diff
@@ -55,6 +74,14 @@ class BaseSSLError(BaseException):
 _CONTAINS_CONTROL_CHAR_RE = re.compile(r"[^-!#$%&'*+.^_`|~0-9a-zA-Z]")
 
 
+_DefaultType = Union[bytes, IO[Any], Iterable[bytes], str]  # Type for HTTP body.
```

Comment:
Let's have a more descriptive global here for HTTP body type

Reply:
Changed to `HTTPBody`